scalability_and_performance/cnf-numa-aware-scheduling.adoc

```diff
@@ -14,8 +14,7 @@ include::snippets/technology-preview.adoc[leveloffset=+1]
 [IMPORTANT]
 .NUMA-aware scheduling Technology Preview limitations and known issues
 ====
-. The NUMA-aware scheduling stack does not support the `container` topology manager scope. For more information, see link:https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#topology-manager-scopes[Topology manager scopes].
-. Alignment of memory resources is not supported. For more information, see link:https://kubernetes.io/docs/tasks/administer-cluster/memory-manager/[Utilizing the NUMA-aware memory manager].
+* Alignment of memory resources is not supported. For more information, see link:https://kubernetes.io/docs/tasks/administer-cluster/memory-manager/[Utilizing the NUMA-aware memory manager].
```

Comment:
you can remove this as well (in another PR), we lifted some limitation with 4.10.3